src/actions_manager.coffee

```diff
@@ -79,7 +79,7 @@ define [
       payload[params.url] = Settings.url.current
       payload[params.referrer] = Settings.url.referrer
       payload[params.shop_code] = @session.shop_code
-      payload[params.metadata] = @session.metadata || ''
+      payload[params.metadata] = @_buildMetadata(JSON.parse(data).sbm)
```

Comment:
in the very first requests "sbm" key might not be present in the json payload

Reply:
indeed, however `_buildMetadata` will return `@session.metadata || ''` in that case (since `sbm` will be `undefined`), which is it's original behaviour